idea/gearexe/frontend/src/features/api/consts.ts

```diff
@@ -0,0 +1,8 @@
+import { LocalStorage, GEAR_API_NODE } from '@/shared/config';
+
+import { getNodeAddressFromUrl } from './utils';
+
+// ! TODO: move to nodesSwitch
+const INITIAL_ENDPOINT = getNodeAddressFromUrl() || (localStorage[LocalStorage.Node] as string | null) || GEAR_API_NODE;
```

Comment:
Let's move it directly. No reason to keep `api` feature, I think it's quite broad and should be shared.